### ISSUE TITLE
Managing connection lost exception

### ISSUE THREAD

**Issue by rafaeldcf** [pyrogram/pyrogram#1448]
### Checklist

- [X] I am sure the error is coming from Pyrogram's code and not elsewhere
- [X] I have searched in the issue tracker for similar bug reports, including closed ones
- [X] I ran `pip3 install -U https://github.com/pyrogram/pyrogram/archive/master.zip` and reproduced the issue using the latest development version

### Description

I have this error sometimes:
```
socket.send() raised exception.
[10] Retrying "updates.GetState" due to: Connection lost
```

How can I manage those exception? Wait more time to reconnect?

### Steps to reproduce

Keep the code running some days

**Comment by sadegh1404** [comment]
Did you find any solution to this? I have the same problem after I upgrade to the latest Layer of telegram API

**Comment by rafaeldcf** [comment]
No, sorry. It seems an error with asynchronous traffic, still investigating it.
